addons/profile-subpages-navigation/addon.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Profile subpages navigation",
+  "description": "Navigate through user profiles with navigation links on sub-pages.",
+  "tags": ["community", "profiles"],
+  "enabledByDefault": false,
+  "versionAdded": "1.25.0",
+  "dynamicDisable": true,
+  "dynamicEnable": true,
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": [
+        "https://scratch.mit.edu/users/*/projects/",
+        "https://scratch.mit.edu/users/*/favorites/",
+        "https://scratch.mit.edu/users/*/studios/",
+        "https://scratch.mit.edu/users/*/studios_following/",
+        "https://scratch.mit.edu/users/*/following/",
+        "https://scratch.mit.edu/users/*/followers/"
+      ]
+    }
+  ],
+  "credits": [
+    {
+      "name": "Xan",
+      "link": "https://scratch.mit.edu/users/-Xanimation-/",
+      "note": "Addon version"
+    },
```

Comment:
Don't forget the `id` property

Reply:
I have updated it to include the `id` property in the credits array: [5eb41c2fde4a833d8a47ad97993b60ac5c6ec8b5](https://github.com/ScratchAddons/ScratchAddons/commit/5eb41c2fde4a833d8a47ad97993b60ac5c6ec8b5)